src/plugins/piral-redux/src/create.test.tsx

```diff
@@ -22,12 +22,16 @@ function createMockContainer() {
   };
 }
 
+const MyComponent: React.FC = () => <div>Component</div>;
```

Comment:
If you only have a single (non-complicated) component like this here, there is no need for JSX / to rename the file. Use `createElement("div", undefined, "Component")` and you are done.